frontend/src/components/widgets/floating-menus/MenuList.vue

```diff
@@ -2,7 +2,7 @@
 	<FloatingMenu class="menu-list" :direction="direction" :type="'Dropdown'" ref="floatingMenu" :windowEdgeMargin="0" :scrollableY="scrollableY" data-hover-menu-keep-open>
 		<template v-for="(section, sectionIndex) in menuEntries" :key="sectionIndex">
 			<Separator :type="'List'" :direction="'Vertical'" v-if="sectionIndex > 0" />
-			<LayoutRow
+			<button
```

Comment:
How is a <label> interactive? Also I can't see any instances where labels or inputs are within the new button elements?

Reply:
`<label>`s are interactive because they can be clicked to cause an input to change (like toggling a checkbox, setting a radio button, etc.). See https://stackoverflow.com/questions/39386497/can-i-nest-button-inside-another-button

Reply:
Oh yes forgot about labels being clickable. I will revert this change and add properly supporting dropdowns with keyboard navigation to issue #575.